aiohttp/typedefs.py

```diff
@@ -23,8 +24,13 @@
     _MultiDictProxy = MultiDictProxy
 
 Byteish = Union[bytes, bytearray, memoryview]
-JSONEncoder = Callable[[Any], str]
-JSONDecoder = Callable[[str], Any]
+JSONObj = Dict[str, Any]
+JSON = Union[
```

Comment:
Technically JSON can contain a single integer, ``None``  or string.
Moreover, it can be **any** python object if there is python -> JSON converter passed by `json.dums(obj, defaul=my_converter)`.
Let's use `Any` for json encoders/decoders.